src/style.css

```diff
@@ -0,0 +1,74 @@
+/* stylelint-disable declaration-block-no-duplicate-properties */
+html {
+  --form-shadow: 0 1px 8px 0 #3d4e611a;
+  --form-shadow-hover: 0 3px 8px 0 #3d4e6133;
+  --color-form: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir, Arial, serif;
+  src: url(./fonts/Avenir-Book.ttf);
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 300px;
+}
```

Comment:
need to fix font connected 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96265844/b13d1be2-ce1d-4a7d-ab53-a722b946c58e)